rosidl_generator_dds_idl/rosidl_generator_dds_idl/__init__.py

```diff
@@ -52,11 +55,76 @@ def generate_dds_idl(
                     },
                     globals=data,
                 )
-                interpreter.file(open(template_file))
+                interpreter.file(open(template_file_msg))
                 interpreter.shutdown()
             except Exception:
                 os.remove(generated_file)
                 raise
+        elif extension == '.srv':
+            srv_spec = parse_service_file(pkg_name, idl_file)
+            default_value_string = None
+            request_fields = [
+                Field(
+                    Type('int64', context_package_name=pkg_name),
+                    'sequence_id', default_value_string),
+                Field(
+                    Type('%sRequest' % srv_spec.srv_name, context_package_name=pkg_name),
+                    'request', default_value_string)
+                ]
+            response_fields = [
+                Field(
+                    Type('int64', context_package_name=pkg_name),
+                    'sequence_id', default_value_string),
+                Field(
+                    Type('%sResponse' % srv_spec.srv_name, context_package_name=pkg_name),
+                    'response', default_value_string)
+                ]
```

Comment:
Should be refactor those two message definitions to be read from `.msg` files instead of constructing them programmatically?